### ISSUE TITLE
Add statistics page alogwith statistics using Chart JS

### ISSUE THREAD

**Issue by arc9693** [lugnitdgp/Hack-Day#70]
Kindly discuss the stats that can be shown before starting to make it.

**Comment by jyarali** [comment]
May you explain more about this?

**Comment by arc9693** [comment]
@jyarali 
We need a page that can show data about this repository in the form of charts.
For ex: 
- One chart can show the number of people who are first-timers in hacktoberfest .
- Another can show the gender ratio

More ideas are welcome.

**Comment by jyarali** [comment]
@arc9693 
Where can I get data(number of first-timers, gender ratio) ?

**Comment by Abha2001** [comment]
i would like to work on this issue

**Comment by arc9693** [comment]
Updates:-
1. The data for charts can be dynamic (using GitHub API) and static as well. Preferably dynamic, but the loading will take time. That's another issue. 
2. Examples of parameters or charts that can be prepared to show the stats:
    - First-time contributors or overall contributors
    - Lines of code added, forks made, issues closed
    - Gender ratio
    - Check out the insights section for inspiration [https://github.com/lugnitdgp/Hack-Day/pulse](https://github.com/lugnitdgp/Hack-Day/pulse)
3. We have to make sure of the correctness of the date range of stats. 2019 should have stats from sep'19-dec'19. Similarly for 2020 stats.
4. Link of the library to be used: [https://www.chartjs.org/](https://www.chartjs.org/). You can use some other library if you want.

@Abha2001 Let me know your take on this.

**Comment by Abha2001** [comment]
okay i will look into the library start working on it
